### PR TITLE
bump version to 12.0.0 Alpha 1 / update readme

### PR DESCRIPTION
**Pull request by mahibi**
Signed-off-by: Marcel Hibbe <dev@mhibbe.de>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,21 +35,29 @@ For more information please review the [guidelines for contributing](https://git
...
-#### Beta Channel
+#### Beta versions (Release Candidates)
 
+##### via Google Play
 Sign up at [Google Play Beta channel](https://play.google.com/apps/testing/com.nextcloud.talk2) to get Release
  Candidates via Google Play.
 
-#### Alpha Channel
+##### via github
+You can also get the Release Candidates at [github releases](https://github.com/nextcloud/talk-android/releases).
```
Comment (mahibi) [inline README.md:47]: fyi @Valdnet

Comment (Valdnet) [reply README.md:47]: Thanks @mahibi 😉.

File: README.md [hunk 2]
```diff
@@ -35,21 +35,29 @@ For more information please review the [guidelines for contributing](https://git
...
 
+#### Alpha versions
+
+##### via Google Play
 To become an alpha tester you have to be signed up for the [Google Play Beta channel](https://play.google.com/apps/testing/com.nextcloud.talk2) 
 and additionally you have to join the [Alpha testing Google Group](https://groups.google.com/g/nextcloud-android-talk-alpha-testing). 
 After that you will receive the alpha versions via the Play Store (initially, this might take some minutes after
- signing up).
-Alpha versions are delivered weekly. Please make sure to remember that these versions might contain bugs and you don't use them in production.
+ signing up). However, in in the Play Store the app will still be named "Nextcloud Talk (Beta)" even if you are an alpha tester, but you will receive the alpha versions.
```
Comment (tobiasKaminsky) [inline README.md:55]: "in in"

**Comment by tobiasKaminsky** [comment]
Strange that Lint is complaining.
Have you maybe merged a PR at some point without lint being satisfied?

Maybe you wan to clean up (in a different PR):
https://www.kaminsky.me/nc-dev/talk-lint/169.html#UnusedResources
and after that rebase this PR.

**Comment by mahibi** [comment]
> Strange that Lint is complaining.
> Have you maybe merged a PR at some point without lint being satisfied?

maybe yesterday 🙈 😉

> 
> Maybe you wan to clean up (in a different PR):
> https://www.kaminsky.me/nc-dev/talk-lint/169.html#UnusedResources
> and after that rebase this PR.

👍 
anyway i won't put too much effort in lint & FindBugs warnings as long master is not merged with master-broken. 
because in master-broken everything would have to be cleaned up again, manually..
does this make sense...?

**Comment by tobiasKaminsky** [comment]
> because in master-broken everything would have to be cleaned up again, manually..
> does this make sense...?

I would have expected that a new feature, that is good for lint / spotbug , would then also be good on master-broken…
But yeah, it might be a bit of double work, if things change between master and master-broken.

**Comment by mahibi** [comment]
> I would have expected that a new feature, that is good for lint / spotbug , would then also be good on master-broken…

so is there a good way to filter the lint / spotbug warnings only for the changes that were made?
preferably already in android studio?
In Android Studio there is "Analyze / Inspect Code ...", but this gives me not the same warnings as in the lint reports.

File: README.md (hunk 2 above)
Comment (Valdnet) [reply README.md]: @tobiasKaminsky You caught it nicely 😉.

**Comment by tobiasKaminsky** [comment]
There is a feature called "baseline", but it never really worked on Files, as there are often too many line changes, which are then considered as new issues, while they were in fact just moved.
My "solution" is just to compare it side by side… :/

**Review by tobiasKaminsky — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (Valdnet) [reply README.md]: @mahibi It remained to fix and Merge 😉.

> signing up). However, in **in** the Play Store the app will still be named "Nextcloud Talk (Beta)" even if you are an alpha tester, but you will receive the alpha versions.

Comment (mahibi) [reply README.md]: @Valdnet "in in" is fixed. Here in the conversations view you still see the old changes (so the "Outdated" label is shown). In the "Files changed" view it is fixed

**Pull request merged**